config.json

```diff
@@ -1680,6 +1680,15 @@
         "prerequisites": ["arrays", "strings", "for-loops", "objects"],
         "difficulty": 3,
         "topics": ["parsing", "strings"]
+      },
+      {
+        "slug": "rest-api",
+        "name": "Rest API",
+        "uuid": "75b4816d-a5d7-4840-92d6-7249c9e8beeb",
+        "practices": [],
+        "prerequisites": ["arrays", "classes", "for-loops", "maps"],
```

Comment:
`maps` is the concept for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
Are you sure this is a prerequisite here? Do you maybe mean `objects` instead (this concept covers using objects as simple key/value collections).

The for-of loop is covered in the concept `array-loops`. Maybe you want to use that here instead of `for-loops`.